### DISCUSSION TITLE
How to resolve type from just type name

### DISCUSSION THREAD

**Discussion by rikashore** [jbevain/cecil#875] [Q&A] [answered]
I'm using Cecil to implement a programming language running on the CLR, I'm trying to import a type reference from just its name such as `Foo` from namespace `Example.Bar`, I have the names of the using statement references (these are not added as assembly references yet) but don't have contextual information of its full qualified name. Example code 
```
use Example.Bar

def ex(foo Foo)
    foo.AProp
```

**Comment by jbevain** [comment, marked as answer]
Hi, There's not much Cecil can do for you here. That's going to be on your compiler to go ahead and try to find types.

**Reply by rikashore** [reply]
Understood, thanks for the reply!
